hw01/hw01_exploring_data.Rmd

```diff
@@ -0,0 +1,15 @@
+---
+title: "hw01_exploring_data"
+author: "Samruddhi Shinde"
+date: "1/24/2021"
+output: html_document
+---
+
+In this document, we will explore the the palmerpenguins dataset.
+
+```{r}
+library(palmerpenguins)
+
+print(penguins)
+```
+
```

Comment:
You may need to add in a couple more functions here other than print. Things like nrow or ncolumn

Reply:
Ignore I saw you added these after I started reviewing.